functions/zoom-downloader.py

```diff
@@ -500,24 +503,8 @@ def zoom_filename(self):
 
             # If the file has been deleted or if the request is not authorized,
             # Zoom will return 200 and an HTML error page.
-            # This is bad, but we have no choice but to try to interpret the
-            # content that Zoom returns to us.
             if "Content-Type" in r.headers and "text/html" in r.headers["Content-Type"]:
-                # Most likely deleted file
```

Comment:
All this code trying to figure out what is going on in the HTML page is too confusing and unreliable so I decided to just get rid of it and simplify things.